app/validators/customer_email_validator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CustomerEmailValidator < ActiveModel::Validator
+  DOMINIOS_PROIBIDOS = ['@gmail.com', '@yahoo.com.br', '@hotmail.com.br', '@iugu-lite.com.br'].freeze
```

Comment:
```suggestion
  BLOCKED_DOMAINS = ['@gmail.com', '@yahoo.com.br', '@hotmail.com.br', '@iugu-lite.com.br'].freeze
```

Aqui passou uns português nessa classe podem dar uma olhada?

Reply:
Feito, alteramos para FORBIDDEN_DOMAINS.